src/clr-angular/data/tree-view/tree.ts

```diff
@@ -14,15 +26,65 @@ import { TREE_FEATURES_PROVIDER, TreeFeaturesService } from './tree-features.ser
     <clr-recursive-children *ngIf="featuresService.recursion"
                             [children]="featuresService.recursion.root"></clr-recursive-children>
   `,
-  providers: [TREE_FEATURES_PROVIDER],
+  providers: [TREE_FEATURES_PROVIDER, TreeFocusManagerService],
+  host: {
+    '[attr.tabindex]': 'tabindex',
+    '[attr.role]': '"tree"',
+    '[attr.aria-multiselectable]': 'isMultiSelectable',
+  },
 })
-export class ClrTree<T> {
-  // This component can also be used just to declare providers once for trees with multiple root nodes.
+export class ClrTree<T> implements AfterContentInit, OnDestroy {
+  constructor(
+    public featuresService: TreeFeaturesService<T>,
```

Comment:
We talked about this, since its only going into master this breaking change of requiring `clr-tree` is acceptable.